build-system/test-configs/config.js

```diff
@@ -98,15 +93,6 @@ const unitTestPaths = [
   'ads/**/test/unit/test-*.js',
   'extensions/**/test/*.js',
   'extensions/**/test/unit/*.js',
-  'src/bento/components/**/test/*.js',
-  'src/bento/components/**/test/unit/*.js',
-];
-
-const bentoUnitTestPaths = [
```

Comment:
I see you're deleting the test-e2e files so shouldn't you delete these too?

Might be cleaner to delete all the test files in a separate PR

Reply:
Yeah nbd as long as you remember to delete the other bento test files.